source/schema/export.txt

```diff
@@ -24,156 +24,95 @@ If you have not already done so, analyze your schema:
    subset of documents from your collection. To learn more about 
    sampling, see :ref:`Sampling <compass-sampling>`.
 
-Once your schema has been analyzed, export your schema:
+Once your schema has been analyzed, export your schema. 
 
-1. In the top menu bar, click :guilabel:`Collection`.
+1. Click :guilabel:`Export Schema`.
 
-#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+   .. figure:: /images/compass/export-schema-button.png
+      :alt: Image showing Export Schema button
 
-.. figure:: /images/compass/share-schema-json.png
-   :alt: Image showing Collection dropdown
+   The :guilabel:`Export Schema` button appears in the top-left corner
+   of the viewport.
 
-Your schema is copied to your clipboard as a |json| object.
+#. In the **Export JSON Schema** modal, select the format to export to.
 
-Schema Object Properties
-------------------------
+   .. figure:: /images/compass/export-modal.png
+      :alt: Export JSON Schema modal
+      :figwidth: 550px
 
-Schema objects have ``count`` and ``fields`` properties:
+   You can export your schema to the following formats:
 
-- ``count`` is an integer that represents the number of documents 
-  :ref:`sampled <compass-sampling>` from the collection to generate the schema.
+   - :ref:`Standard <export-format-standard>` format.
```

Comment:
[suggestion] [style guide](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items) - since these list items are fragments, I don't think we need the punctuation at the end!

Reply:
Thanks -- updated.